doc/factory/configuration.html

```diff
@@ -578,43 +578,53 @@ <h3 class="western"><a name="args_global"></a>Global arguments</h3>
-          <li>
-            <a name="log_retention" />
+          <li id="log_retention">
             <a name="process_logs" />
             <div class="xml">
               &lt;glidein&gt;&lt;log_retention&gt;&lt;process_logs&gt;&lt;process_log
               max_days=&quot;<i>max days</i>&quot; min_days=&quot;<i>min days</i
               >&quot; max_mbytes=&quot;<i>max mega bytes</i>&quot;
-              type=&quot;<i>INFO</i>&quot; backup_count=&quot;<i>backup count</i
+              msg_types=&quot;<i>INFO</i>&quot; backup_count=&quot;<i
+                >backup count</i
               >&quot; compression=&quot;<i>gz</i>&quot;/&gt;
             </div>
             <p>
               The admin can configure one or more logs with any combination of
-              following log message types:
+              following log message types. Types are converted to uppercase,
+              unknown typer are silently ignored, the default is INFO,ERR,WARN:
             </p>
             <ul>
               <li>
-                INFO: Informational messages about the state of the system.
+                INFO: (INFO level) Informational messages about the state of the
+                system.
+              </li>
+              <li>
+                DEBUG: Debug message (DEBUG level). These are additional
+                informational messages that describe code execution in detail.
+              </li>
+              <li>
+                ERR: Error messages (ERROR and CRITICAL levels). These may
+                include tracebacks.
               </li>
               <li>
-                DEBUG: Debug message. These are additional informational
-                messages that describe code execution in detail.
+                WARN: Warning messages (WARNING level). These warn of conditions
+                that were found that don't necessarily cause abnormal execution.
               </li>
-              <li>ERR: Error messages. These may include tracebacks.</li>
               <li>
-                WARN: Warning messages. These warn of conditions that were found
-                that don't necessarily cause abnormal execution.
+                ALL: Same as INFO,DEBUG,ERR,WARN. Other message type keywords
+                are ignored when ALL is present.
               </li>
               <li>
                 ADMIN: This is a special setting only applicable to Factory logs
                 (and not entry logs) that contains admin messages, such as
-                reconfig comments. This will create a new log called
-                factoryadmin.extension.log. Currently, it cannot be combined
-                with other message types.
+                reconfig comments. This will create a new log appending "admin"
+                to the name, e.g. factoryadmin.extension.log. Currently, it
+                cannot be combined Other message type keywords are ignored.
+                INFO, DEBUG, ERR, and WARN massages are added to the log.
```

Comment:
The new version seems to have removed the second part of the last statement. Is that intended?